src/picasso-definition/interactions/tap-mini-chart/index.js

```diff
@@ -0,0 +1,31 @@
+import KEYS from '../../../constants/keys';
+import updateDataView from './update-data-view';
+
+const tap = ({ eventName = 'tap-mini-chart', viewHandler }) => {
+  let targetComponents;
```

Comment:
This variable is not really needed, can return directly from the `enable` function.